pkg/cmd/extension/manager.go

```diff
@@ -223,7 +223,7 @@ func (m *Manager) getCurrentVersion(extension string) string {
 	}
 	dir := m.installDir()
 	gitDir := "--git-dir=" + filepath.Join(dir, extension, ".git")
-	cmd := m.newCommand(gitExe, gitDir, "rev-parse", "HEAD")
+	cmd := m.newCommand(gitExe, gitDir, "rev-parse", "--short", "HEAD")
```

Comment:
Rather than retrieve the short SHA from git I think it would be better to continue to retrieve the full SHA and have the place that needs to display the current version do any formatting that is required. In our case it would be truncating it to 8 characters.